doc/10-icinga-template-library.md

```diff
@@ -2259,6 +2259,42 @@ snmp_service_count         | **Optional.** Compare matching services with a spec
 snmp_service_showall       | **Optional.** Show all services in the output, instead of only the non-active ones. Defaults to false.
 snmp_service_noregexp      | **Optional.** Do not use regexp to match NAME in service description. Defaults to false.
 
+### Elasticsearch Netways <a id="plugin-check-command-elasticsearch-netways"></a>
+
+This category includes all subcommands using the elasticsearch-netways plugin.
```

Comment:
You don't actually call the plugin this way, do you?

https://github.com/NETWAYS/check_elasticsearch/search?q=elasticsearch-netways